cognite/powerops/resync/config/production/plant.py

```diff
@@ -64,6 +65,7 @@ def from_cdf_resources(cls, asset: Asset, relationships: list[Relationship], **k
             p_min=float(asset.metadata["p_min"]),
             p_max=float(asset.metadata["p_max"]),
             head_loss_factor=float(asset.metadata["head_loss_factor"]),
+            connection_losses=json.loads(asset.metadata.get("connection_losses") or "{}"),
```

Comment:
Hmm, what happens here? Json.loads- but shouldn't connection_losses just be a float? Does it make sense to have this structure then? 

E.g. for head_loss_factor is loads a float only.